### PR TITLE
B.E.P.I.S. Version 2.0: Cargo Trimming

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

So, the B.E.P.I.S. after some experimentation and forethought was in a bad place. Thanks to the power of _STATS AND TREND DATA_, the balance around the B.E.P.I.S. was in a truly trash garbage place:
image
Here's a vetted dataset based off of shifts with >25 players in a single shift, showing the end-shift average cash on crewmembers accounts introduced into the round. In the largest sample, So: We using the generalized data (Average player incomes, Average Correlary costs for an "end-game costing object", I shifted the whole reward spectrum of the BEPIS to make it an actual attainable goal for players and an effective time-gated system for both economy and R&D.

Second, I moved BEPIS from Science to Cargo, in order to help the crew with the largest focus on acquisition of wealth work on one of the most expensive station projects available every shift. Plus there's another way that cargo and science and leverage their mutual needs, as Cargo needs Science to cash out and claim the tech disk they're earning from player funds.

Finally, this trims out the Rolling Table and Mauna Mug tech nodes into one, in order to help consolidate the reward pool and to help keep BEPIS nodes to 2 rewards+ per tech, as was ordained by me because it provides more value for the player's efforts.

## Why It's Good For The Game

It feels more in-line with cargo's gameplay loop than science's all things considered, plus if the data and guesstimations I've done are close enough it should make the BEPIS content more accessible to the crew. In addition to the math changes, the QOL for this by adjusting this over to cargo will hopefully provide a small bookmark into the cargo gameplay loop between "Shuffle crates" and "order guns" which we're all too accustomed to.

## Changelog
:cl:
tweak: The BEPIS has been moved to the Cargo Bay from Science, and it's prices have been adjusted to roughly late-game proportions for most players.
balance: The Rolling Tables and Mauna Mug BEPIS techs have been merged into one.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Higher chance for the best item in the game, nothing else matters

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1078,23 +1078,12 @@
...
 	experimental = TRUE
 
-/datum/techweb_node/rolling_table
-	id = "rolling_table"
-	display_name = "Advanced Wheel Applications"
-	description = "Adding wheels to things can lead to extremely beneficial outcomes."
+/datum/techweb_node/extreme_office
+	id = "extreme_office"
+	display_name = "Advanced Office Applications"
+	description = "Some of smartest lab guys got together on a friday and improved office efficiency by 350%. Here's how."
```
Comment (ATH1909) [inline code/modules/research/techweb/all_nodes.dm:1084]:
```suggestion
	description = "Some of smartest lab guys got together on a Friday and improved office efficiency by 350%. Here's how."
```

**Comment by ExcessiveUseOfCobblestone** [comment]
> doesnt put r2 value

cringe

Also does the round-end average really matter? It should be based on the total amount the crew has since ideally people (not really 1 individual) should be investing in the machine for "global" benefits.

**Comment by Tlaltecuhtli** [comment]
what does this graphic even mean? you should probably only count cash deltas by removing paychecks, as 90% of people usually just never interact with money or buy like 1 crowbar from vendor, to get active money generated/ exchanged

**Comment by Tharcoonvagh** [comment]
Why cargo? You cant remove money from the budget card so what's the point in moving it

**Comment by ArcaneMusic** [comment]
> Why cargo? You cant remove money from the budget card so what's the point in moving it

You have to physically use an ID card on the BEPIS in order to add money to it, so that's a moot point.



> Also does the round-end average really matter? It should be based on the total amount the crew has since ideally people (not really 1 individual) should be investing in the machine for "global" benefits.

Now that the big econ PR is merged I'll want to do some toying around to make sure my data's on track with my data's results, so I'll compile some more flexible data to compensate

**Comment by kriskog** [comment]
Merge conflict

**Review by kriskog — DISMISSED** [review]

**Review by kriskog — APPROVED** [review]

**Comment by griffithlol** [comment]
Dunno how to quote, but you shouldn't be thinking in terms of ideally. You should be thinking in terms of what's likely

**Comment by kopoba** [comment]
moving bepis to cargo is so strange like what they will do with discs? trash them

**Pull request merged**